mypy/util.py

```diff
@@ -476,6 +468,13 @@ def hash_digest(data: bytes) -> str:
     return hashlib.sha256(data).hexdigest()
 
 
+def parse_gray_color(cup: bytes) -> str:
+    """Reproduce a gray color in ANSI escape sequence"""
+    set_color = ''.join([cup[:-1].decode(), 'm'])
+    gray = curses.tparm(set_color.encode('utf-8'), 1, 89).decode()
+    return gray
```

Comment:
This seems quite odd: "cup" is for "move to row/column", replacing its `H` with `m` in the end seems quite hackish.
Why not use `\e[90m` directly?
Ref: https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters